.circleci/test-api.sh

```diff
@@ -0,0 +1,8 @@
+#! /usr/bin/env bash
+
+PORT=5000
+ENDPOINT=spec
+
+openfisca serve --country-package openfisca_country_template --port $PORT &
+
+curl --retry-connrefused --retry 10 --retry-delay 5 --fail http://127.0.0.1:$PORT/$ENDPOINT | python -m json.tool > /dev/nul
```

Comment:
```suggestion
curl --retry-connrefused --retry 10 --retry-delay 5 --fail http://127.0.0.1:$PORT/$ENDPOINT | python -m json.tool > /dev/null
```

Reply:
This does work because `curl --fail` will not forward the (potentially valid JSON) response, as evidenced by:

```
› curl --header 'Accept:application/json' --retry-connrefused --retry 1 --retry-delay 1 --fail https://httpstat.us/418 | python -m json.tool
curl: (22) The requested URL returned error: 418 
No JSON object could be decoded
```